mmdet/models/backbones/hrnet.py

```diff
@@ -0,0 +1,503 @@
+import logging
+
+import torch.nn as nn
+from mmcv.cnn import constant_init, kaiming_init
+from mmcv.runner import load_checkpoint
+
+from ..registry import BACKBONES
+from ..utils import build_norm_layer, build_conv_layer
+from .resnet import BasicBlock, Bottleneck
+
+
+class HighResolutionModule(nn.Module):
```

Comment:
May just use the abbreviation `HRModule` (since the net is renamed to HRNet) and add a docstring.